IS7024Project/Pages/Index.cshtml.cs

```diff
@@ -28,17 +29,16 @@ public void OnGet()
             {
                 //Consuming Parks data
                 string parksJSON = webClient.DownloadString("https://raw.githubusercontent.com/JMFrank215/IS7024Project/master/Parks_Data.txt");
-                var park = Parks.FromJson(parksJSON);
-                ViewData["Parks"] = park;
+                var parks = Parks.FromJson(parksJSON);
```

Comment:
changed park to parks since the variable parks represents an array of multiple parks.

Reply:
Good idea.